.gitignore

```diff
@@ -23,6 +23,8 @@
 /storage/*
 !/storage/.keep
 
+/db/schema.rb
```

Comment:
You added file to gitignore, but it's still in the repo.

Reply:
Ah Shit, Here We Go Again